packages/react-native/Libraries/NewAppScreen/components/LearnMoreLinks.js

```diff
@@ -74,10 +74,10 @@ const links = [
   },
   {
     id: 9,
-    title: 'Follow us on Twitter',
-    link: 'https://twitter.com/reactnative',
+    title: 'Follow us on X',
```

Comment:
nit: Perhaps for text wrapping and futureproofing, can we simplify this left-hand side heading to "Follow us"? :)

Reply:
Done :)